packages/components/src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -14,21 +48,35 @@ const StyledIndicator = styled.div`
   height: 100%;
   width: 100%;
   transition: transform 660ms cubic-bezier(0.65, 0, 0.35, 1);
-  border-radius: ${theme.radii.borderRadius10};
-  background-color: ${theme.colors.colorBackgroundPrimaryWeak};
+  border-radius: ${(props: StyledIndicatorProps) =>
+    get(theme.radii, String(props.radius), theme.radii.borderRadius10)};
+  background-color: ${(props: StyledIndicatorProps) =>
+    get(
+      theme.colors,
+      String(props.background),
+      theme.colors.colorBackgroundPrimaryWeak
+    )};
 `;
 
-export interface ProgressBarProps extends Omit<ProgressProps, "value"> {
-  /** The progress value. */
-  value: number;
-}
-
 /** Displays an indicator showing the completion progress of a task, typically displayed as a progress bar. */
 const ProgressBar = React.forwardRef<HTMLDivElement, ProgressBarProps>(
-  ({ value, ...props }, ref) => (
-    <StyledProgress as={Root} ref={ref} value={value} {...props}>
+  (
+    { value, h, indicatorColor, backgroundColor, borderRadius, ...props },
+    ref
+  ) => (
+    <StyledProgress
+      as={Root}
+      background={backgroundColor}
+      h={h}
+      radius={borderRadius}
+      ref={ref}
+      value={value}
+      {...props}
+    >
       <StyledIndicator
         as={Indicator}
+        background={indicatorColor}
```

Comment:
I used the background name instead of `indicatorColor` because it's easier to think about it from the perspective of the indicator/progress but also to avoid this error:

<img width="1638" alt="Screenshot 2022-12-09 at 13 21 51" src="https://user-images.githubusercontent.com/55699538/206709072-190621ba-14d1-499c-ade4-3095c367fb08.png">